packages/markdown-docx/src/OOXMLTransformer/index.js

```diff
@@ -58,171 +87,178 @@ class OoxmlTransformer {
     }
 
     /**
-     * Gets the value of the variable
+     * Constructs a ciceroMark Node JSON from the information
      *
-     * @param {Array} elements the variable elements
-     * @returns {string} the value of the variable
+     * @param {object} nodeInformation Contains properties and value of a node
+     * @return {object} CiceroMark Node
      */
-    getValue(elements) {
-        const variableContents = elements[1];
-        for (const property of variableContents.elements[0].elements) {
-            if (property.name === 'w:t') {
-                return property.elements[0].text;
-            }
+    construtCiceroMarkNodeJSON(nodeInformation) {
+        let obj = {};
+        if (nodeInformation.nodeType === 'softbreak') {
+            obj = {
+                $class: `${NS_PREFIX_CommonMarkModel}Softbreak`,
+            };
+        } else if (nodeInformation.nodeType === 'variable') {
+            obj = {
+                $class: `${NS_PREFIX_CiceroMarkModel}Variable`,
+                value: nodeInformation.value || 'Not provided',
```

Comment:
If the value of the variable is `''`, you transformer will return `'Not provided`. So I think you can omit the `or` condition.

Moreover, I don't think so a variable entity would be without these attributes ever so you should omit all the `or` conditions. However, if this was possible `CiceroMark` should have defined a default _empty_ value for each of these attributes.

Reply:
This was done in the case that maybe a user uses sdt in word first and then tries to convert it in a `ciceroMark`. Although it won't be a variable and I agree here so I will remove this.